### ISSUE TITLE
[Paper Note] DP-RFT: Learning to Generate Synthetic Text via Differentially Private Reinforcement Fine-Tuning, Fangyuan Xu+, arXiv'26, 2026.02

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#4779]
# URL
- https://arxiv.org/abs/2602.18633
# Authors
  - Fangyuan Xu
  - Sihao Chen
  - Zinan Lin
  - Taiwei Shi
  - Sydney Graham
  - Pei Zhou
  - Mengting Wan
  - Alex Stein
  - Virginia Estellers
  - Charles Chen
  - Morris Sharp
  - Richard Speyer
  - Tadas Baltrusaitis
  - Jennifer Neville
  - Eunsol Choi
  - Longqi Yang
# Abstract
  - Differentially private (DP) synthetic data generation plays a pivotal role in developing large language models (LLMs) on private data, where data owners cannot provide eyes-on access to individual examples. Generating DP synthetic data typically involves a difficult trade-off. On one hand, DP finetuning methods train an LLM as a synthetic data generator with formal privacy guarantees, yet it still requires the raw content of private examples for model training. However, methods that avoid direct exposure to private data are bounded by an off-the-shelf, un-finetuned model, whose outputs often lack domain fidelity. Can we train an LLM to generate high-quality synthetic text without eyes-on access to individual private examples? In this work, we introduce Differentially Private Reinforcement Fine-Tuning (DP-RFT), an online reinforcement learning algorithm for synthetic data generation with LLMs. DP-RFT leverages DP-protected nearest-neighbor votes from an eyes-off private corpus as a reward signal for on-policy synthetic samples generated by an LLM. The LLM iteratively learns to generate synthetic data to maximize the expected DP votes through Proximal Policy Optimization (PPO). We evaluate DP-RFT for long-form and domain-specific synthetic data generation, such as news articles, meeting transcripts, and medical article abstracts. Our experiments show that DP-RFT closes the gap between private evolution and DP finetuning methods in terms of the fidelity and downstream utility of the generated synthetic data, while respecting the private data boundary.
# Translation (by gpt-5-nano)
- 差分プライバシー（DP）保護された合成データ生成は、個々の例をデータ所有者が直接確認できないプライベートデータ上で大規模言語モデル（LLMs）を開発する際に、極めて重要な役割を果たします。DP合成データの生成には通常、難しいトレードオフが伴います。 一方、DP微調整法は正式なプライバシー保証を伴う合成データ生成器としてLLMを訓練しますが、モデル訓練にはプライベートな例の生データの内容が依然として必要です。 しかし、個人データへの直接露出を回避する方法は、市販の未微調整モデルに制約され、その出力はしばしばドメイン適合性を欠きます。 個々のプライベートな例を直接確認できない状況で、高品質な合成テキストを生成するLLMを訓練できるだろうか。 本研究では、Differentially Private Reinforcement Fine-Tuning（DP-RFT）を導入します。これはLLMを用いた合成データ生成のためのオンライン強化学習アルゴリズムです。 DP-RFT は、個人データを直接参照せずにアクセスされるプライベートコーパスから得られるDP保護済み最近傍投票を、オンポリシーの合成サンプルの報酬信号として活用します。 LLM は Proximal Policy Optimization（PPO）を通じて、期待されるDP投票を最大化するように合成データの生成を反復的に学習します。 長文およびドメイン特化の合成データ生成、例えばニュース記事、会議の議事録、医療論文の抄録を対象に DP-RFT を評価します。 実験の結果、生成された合成データの忠実度と下流の有用性の観点で、プライベートデータの境界を尊重しつつ、プライベートデータを用いた進化的学習とDP微調整法とのギャップを縮小することを示しました。
# Summary (by gpt-4o-mini)
- DP-RFTを用いて、プライベートデータに直接アクセスせずに合成データを生成するためのオンライン強化学習アルゴリズムを提案。合成サンプルの報酬信号にDP保護済み最近傍投票を活用し、LLMが期待されるDP投票を最大化するよう学習。長文やドメイン特化のデータ生成において、プライベートデータの境界を尊重しつつ、従来の手法とのギャップを縮小することに成功。

**Comment by AkihikoWatanabe** [comment]
元ポスト:https://x.com/brunchavecmoi/status/2028825108542783839?s=46&t=Y6UuIHB0Lv0IpmFAjlc2-Q

**Comment by AkihikoWatanabe** [comment]
プライベートなデータの保有者が差分プライバシーが保護された状態でLLMのロールアウトに対してvotingによるrewardを返せば、個別のLLMはプライバシーに保護されたデータを見なくてもvotingによるスコアが最大となるように学習できるというアイデア。これによりプライバシーによる課題によりデータがオープンにならないドメインでも、この枠組みでLLMをpost-trainingすれば、LLMが合成データの生成器として振舞えるため、プライベートなドメインのデータスケーラビリティの課題の解決につながるのではという提案

**Comment by AkihikoWatanabe** [comment]
これは利用規約などで個人情報の扱いに関して何らかのユーザとの取り決めがあった場合、どういう扱いになるのだろうか。
Gemini Proに質問して得た感想としては、少なくとも差分プライバシーによってreward signalが個人情報を含むデータではないと保証されたとしても（プライバーバジェットがどの程度設定されていれば問題ないのかといった合意があるかと言われると怪しいらしい）、reward signalを計算する部分においては個人情報を含むデータを活用しているため、個人情報利用のスコープにそれが許容されるようなステートメントが入っていないと、こういった手法を実施することは無理なのかもしれない。
